tools/release-post-generator/lib/logger.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import ora, { Ora } from 'ora';
+import chalk from 'chalk';
+import { getEnvVar } from './environment';
+
+const { log, error, warn } = console;
+export class Logger {
```

Comment:
basic logger class, includes `ora` for progress bars and `chalk` for colorizing output. Since Commander doesn't come with anything like this you need to DIY. 

I think it would make sense to reuse this for the other cli tools as we move them to Commander.

Reply:
>I think it would make sense to reuse this for the other cli tools as we move them to Commander.

Yes please!